README.md

```diff
@@ -145,6 +145,9 @@ All [watcher](#watcher) & [sender](#sender) services launch when `docker-compose
 * `chdir` to the directory containing the `docker-compose.yml` file used to run the bridge instance
 * [View the logs](https://docs.docker.com/v17.09/compose/reference/logs/) : `docker-compose logs`
 
+**Note 2**: 
```

Comment:
Just put these sentences without `Note 2` prior to previous `Note`. I think this information is general enough so could be located there.

Reply:
Thank you @akolotov .
Just updated with a new commit. Let me know if I correctly understood what you meant.

Reply:
thanks!